api/types/sortby.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import (
+	"strings"
+)
+
+func GetSortByFromString(sortStr string) SortBy {
+	var sortBy SortBy
```

Comment:
Nit: What I've learned recently is that naming the return values implicitly instantiates them, meaning that this code could be replaced with:

```suggestion
func GetSortByFromString(sortStr string) (sortBy SortBy) {
```